### PR TITLE
Next project setup with styled-components

### PR DESCRIPTION
**Pull request by mashish584**

### CODE REVIEW

File: client/styles/Home.module.css [hunk 1]
```diff
@@ -0,0 +1,121 @@
+.container {
+  min-height: 100vh;
+  padding: 0 0.5rem;
+  display: flex;
```
Comment (ilakshay14) [inline client/styles/Home.module.css:4]: Separate out commonly used styles and group them.

Use styled components to their full extent - extending css, existing styled components, etc.

File: client/styles/Home.module.css [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+  font-size: 4rem;
+}
+
+.title,
+.description {
+  text-align: center;
+}
+
+.description {
+  line-height: 1.5;
```
Comment (ilakshay14) [inline client/styles/Home.module.css:59]: unit?

File: client/styles/Home.module.css [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  flex: 1;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.footer {
+  width: 100%;
+  height: 100px;
```
Comment (ilakshay14) [inline client/styles/Home.module.css:22]: suggestion: use rem.

File: client/styles/Home.module.css [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+  font-size: 1.25rem;
+  line-height: 1.5;
+}
+
+.logo {
+  height: 1em;
+  margin-left: 0.5rem;
+}
+
+@media (max-width: 600px) {
```
Comment (ilakshay14) [inline client/styles/Home.module.css:116]: Use rem.

File: client/styles/globals.css [hunk 5]
```diff
@@ -0,0 +1,16 @@
+html,
+body {
+  padding: 0;
+  margin: 0;
+  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
```
Comment (ilakshay14) [inline client/styles/globals.css:5]: do we really need to inject so many fonts? 
I would say add 1 of your choice and 1 default.

**Review by ilakshay14 — COMMENTED** [review]
Check out the suggestions.

- use styled components' modularity. dont rewrite css again and again. Create extensions.
- use rem instead of px.

File: client/styles/globals.css (hunk 5 above)
Comment (mashish584) [reply client/styles/globals.css:5]: @ilakshay14  These are default styling of next-boilerplate. I'll be removing all default styles in next update.

**Pull request merged**